adds note about package-lock files (#276)
springernature/frontend-playbook: practices/managing-node-projects.md

Better is to use a version of node that ships with `npm` version 5.7.0 or higher ([node v10.3+ or 8.12+](https://nodejs.org/en/download/releases/)). This is because `npm` 5.7.0+ supports the `ci` argument, which is good because **`npm ci` will always install predictably**, using the `package-lock.json` as a source of authority.

> [`npm ci` is also much quicker than `npm install`](https://docs.npmjs.com/cli/ci.html#description) if the `./node_modules` directory is not present (such as in a Ci environment).

Reviewer: ```suggestion
[`npm ci` is also much quicker than `npm install`](https://docs.npmjs.com/cli/ci.html#description) if the `./node_modules` directory is not present (such as in a CI environment).
```

capitalises acronym

(You could use `npm ci` by specifing a newer version of `npm` than is recommended for your [particular version of node](https://nodejs.org/en/download/releases/), but `npm` is itself written in node and [only supports certain node versions](https://github.com/npm/cli/blob/latest/lib/utils/unsupported.js).)

#### Automatically running `nvm use`

To avoid accidentally forgetting to run `nvm use` before working on a node project, there are useful shell extensions which will do it for you
([zsh](https://github.com/creationix/nvm#calling-nvm-use-automatically-in-a-directory-with-a-nvmrc-file),
[bash](https://stackoverflow.com/questions/23556330/run-nvm-use-automatically-every-time-theres-a-nvmrc-file-on-the-directory)),
which are well worth installing!

### Should I check in `package-lock.json` to version control?

> If using a version of `npm` that supports the `ci` argument, yes you should, for two main reasons:

Reviewer: ```suggestion
Yes, if you're using a version of `npm` that supports the `ci` argument. There are two main reasons:
```

Don't tease us by hiding the answer at the end of the sentence 😉
